streaming/src/main/scala/io/gearpump/streaming/appmaster/ExecutorManager.scala

```diff
@@ -121,9 +121,12 @@ private[appmaster] class ExecutorManager(
   def terminationWatch : Receive = {
     case Terminated(actor) =>
       val executorId = Try(actor.path.name.toInt)
-      LOG.error(s"Executor $executorId is down")
       executorId match {
-        case scala.util.Success(id) => taskManager ! ExecutorStopped(id)
+        case scala.util.Success(id) => {
+          executors -= id
+          LOG.error(s"Executor $id is down")
```

Comment:
Not big thing, but I'd like to get clear. `LOG.error` means the executor is terminated **unexpectedly**. Expected behavior should be logged with `INFO`.